src/azul/indexer/__init__.py

```diff
@@ -403,6 +408,27 @@ def drs_path(self, manifest_entry: JSON) -> Optional[str]:
         """
         raise NotImplementedError
 
+    def _reject_joiner(self, value: Union[JSON, JSONs, PrimitiveJSON]):
```

Comment:
The declared type doesn't cover arrays of primitive values:

```suggestion
    def _reject_joiner(self, value: AnyJSON):
```